core/src/main/java/org/apache/accumulo/core/clientImpl/bulk/BulkImport.java

```diff
@@ -229,6 +228,7 @@ private Path checkPath(FileSystem fs, String dir) throws IOException, AccumuloEx
   }
 
   @Override
+  @Deprecated(since = "2.1.0")
   public ImportMappingOptions executor(Executor service) {
```

Comment:
It's not clear. Why is this option being deprecated in the bulk import builder?

Reply:
It's available in the ClienttThreadPools implementation.

Reply:
Okay, a javadoc deprecated note to point to the new API would be helpful here

Reply:
Resolved in 6c996df